src/Sylius/Bundle/ShopBundle/Twig/Component/Product/CountReviewsComponent.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ShopBundle\Twig\Component\Product;
+
+use Sylius\Component\Channel\Context\ChannelContextInterface;
+use Sylius\Component\Core\Model\ChannelInterface;
+use Sylius\Component\Core\Model\Product;
+use Sylius\Component\Core\Model\ProductReview;
+use Sylius\Component\Core\Repository\ProductReviewRepositoryInterface;
+use Sylius\Component\Locale\Context\LocaleContextInterface;
+use Symfony\UX\TwigComponent\Attribute\AsTwigComponent;
+use Symfony\UX\TwigComponent\Attribute\ExposeInTemplate;
+
+#[AsTwigComponent]
+class CountReviewsComponent
```

Comment:
`ReviewsComponent` and `CountReviewsComponent` can be one component imo 🤔

Reply:
ReviewsComponent is used on Reviews index to show all the reviews, CountReviewsComponent on product show summary to just show the number of reviews, can I somehow use 2 different templates with one component?

Reply:
Just add in the component

```php
public bool $showRatingsCount = true
```
and then:

```
component('component', { showRatingsCount: false })
```